datanode/gateway/graphql/server.go

```diff
@@ -23,15 +23,18 @@ import (
 	"strconv"
 	"time"
 
+	"golang.org/x/crypto/acme/autocert"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/connectivity"
```

Comment:
It's just style, but usually we try to have imports with the following order in the codebase:
- std import
- project import (code.vegaprotocol.io/*)
- external dependencies
Each block of imports being separated with a blank line.